app.js

```diff
@@ -1,11 +1,13 @@
 // IMPORT PACKAGES
 // Here you should import the required packages for your Express app: `express` and `morgan`
+const express= require('express')
 
+const morgan = require('morgan')
 
```

Comment:
All the requieres by good practice should be at the top of the file, in this way we can know which package or files are imported to our file 👍

```suggestion
const projects = require('./data/projects.json')
const articles = require('./data/articles.json')
```